### PR TITLE
Exclude migration file from Metrics/AbcSize check

### PR DESCRIPTION
**Pull request by magni-**
First of many smaller PRs from #752

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -3,9 +3,7 @@
 
 # Offense count: 19
 Metrics/AbcSize:
-  Exclude:
-    - 'test/dummy/db/migrate/20110208155312_set_up_test_tables.rb'
-  Max: 46 # Goal: 15
+  Max: 50
```
Comment (jaredbeck) [inline .rubocop_todo.yml]: I don't understand why the `Max` increased here. `Max` should either stay the same, or decrease.

Comment (magni-) [inline .rubocop_todo.yml]: Whoops, looks like a bad rebase, I'll fix :+1:

Comment (magni-) [inline .rubocop_todo.yml]: Fixed

**Comment by jaredbeck** [comment]
Thanks!

**Pull request merged**